api-server/server/passport-providers.js

```diff
@@ -29,8 +29,8 @@ export default {
     clientID,
     clientSecret,
     domain,
-    cookieDomain: 'freeCodeCamp.org',
-    callbackURL: '/auth/auth0/callback',
+    cookieDomain: process.env.COOKIE_DOMAIN || 'localhost',
```

Comment:
Let's make this environment var to be called as such, if we are forced to use the absolute URL.